format/mbr/mbr.go

```diff
@@ -0,0 +1,22 @@
+package mbr
+
+import (
+	"github.com/wader/fq/format"
+	"github.com/wader/fq/format/registry"
+	"github.com/wader/fq/pkg/decode"
+)
+
+func init() {
+	registry.MustRegister(decode.Format{
+		Name:        format.MBR,
+		Description: "Master Boot Record",
+		DecodeFn:    mbrDecode,
+	})
+}
+
+func mbrDecode(d *decode.D, in interface{}) interface{} {
+	d.FieldRawLen("code_area", 446*8)
```

Comment:
Plan is to use some kind of x86_16 decoder here?

Reply:
For the first iteration I'm going to focus on the partition table. Your work on the x86_16 decoder can be reused here.

Reply:
Sounds good 👍